swagconnect/github/connectors.py

```diff
@@ -5,6 +5,7 @@
 
 class GithubConnector(CustomOAuth2Adapter):
     id = 'github'
+    client_class = CustomOAuth2Adapter
```

Comment:
I think this is wrong, because `GithubConnector` is derived from the `CustomOuath2Adapter` which already has `client_class=OAuthClient`